addon/services/tour.js

```diff
@@ -154,60 +168,21 @@ export default Service.extend(Evented, {
       //Style all internal elements as well
       const children = $(currentElement).children();
       const clonedChildren = highlightElement.children();
+
       for (let i = 0; i < children.length; i++) {
-        clonedChildren[i].style.cssText = window.getComputedStyle(children[i]).cssText;
+        clonedChildren[i].style.cssText =
+          window.getComputedStyle(children[i]).cssText;
       }
-      highlightElement.css('position', 'absolute');
-      highlightElement.css('left', elementPosition.left);
-      highlightElement.css('top', elementPosition.top);
-      highlightElement.css('width', elementPosition.width);
-      highlightElement.css('height', elementPosition.height);
-      highlightElement.css('z-index', '10002');
-    }
-  },
 
-  /**
-   * Exchanges the given attachment configuration for the object equivalent with
-   * the DOM element passed in directly.
-   *
-   * Allows for use of pseudoselectors, because it uses jQuery's selector engine
-   * instead of the built-in one
-   *
-   * @param {Object} attachTo The given "attachTo" configuration
-   * @return {Object} The optimized configuration
-   * @private
-   */
-  exchangeForAttachmentConfig(attachTo) {
```

Comment:
This is likely because the dummy app only uses a string for attachTo. Try using the object form of attachTo in the dummy app and/or write a test that tries using both. It needs that method to convert it.